init-tools.cmd

```diff
@@ -83,7 +83,7 @@ if not [%INIT_TOOLS_ERRORLEVEL%]==[0] (
 
 :: Restore a custom RoslynToolset since we can't trivially update the BuildTools dependency in CoreRT
 echo Configurating RoslynToolset...
-set ROSLYNCOMPILERS_VERSION=3.1.0-beta3-19213-02
+set ROSLYNCOMPILERS_VERSION=3.2.0-beta1-19229-02
```

Comment:
Could you please bump BUILD_TOOLS_SEMAPHORE ?

Reply:
Fixed. Also added a comment at each point indicating the semaphore/other locations that need updating so that I hopefully remember to do this in the future.